README.md

```diff
@@ -72,6 +93,19 @@ More [Examples]
 |Android|🚧 Not yet|
 |Web|🚧 Not yet|
 
+## Love Ribir?
+
+If you like Ribir, give our repo a [⭐ STAR ⬆️](https://github.com/RibirX/Ribir) and [WATCH 👀](https://github.com/RibirX/Ribir/subscription) our repository to stay updated with the latest developments!
+
+Every encouragement and feedback can support us to go further.
+
+## Known Issues
+
+The most important issue we are facing is that `pipe/watch` may easily introduce a cycle reference, which will cause a memory leak. We'll resolve this issue in `v0.2.0`, and then, in most cases, the framework will be able to avoid cycle reference automatically, documentation will be updated to explain how to avoid cycle reference in the rest cases, and a good API will be provided to help developers resolve it.
```

Comment:
The `pipe/watch` may easily introduce a cycle reference, which will cause a memory leak. We'll resolve this issue in v0.2.0. 

In most cases, the framework will be able to avoid cycle reference automatically. Documentation will be updated to explain how to avoid cycle reference in the rest cases, and we will provide an API to help developers to resolve it.